atlassian_jwt_auth/tests/aio/test_public_key_provider.py

```diff
@@ -0,0 +1,68 @@
+import aiohttp
+from asynctest import TestCase, CoroutineMock
+from multidict import CIMultiDict
+
+from atlassian_jwt_auth import aio
+from atlassian_jwt_auth.key import PEM_FILE_TYPE
+from atlassian_jwt_auth.tests import utils
+
+
+class DummyHTTPSPublicKeyRetriever(aio.HTTPSPublicKeyRetriever):
+    def set_headers(self, headers):
+        self._session.get.return_value.headers.update(headers)
+
+    def set_text(self, text):
+        self._session.get.return_value.text.return_value = text
+
+    def _get_session(self):
+        session = CoroutineMock(spec=aiohttp.ClientSession).return_value
+        resp = session.get.return_value
+        resp.headers = CIMultiDict({"content-type": PEM_FILE_TYPE})
+        resp.text = CoroutineMock(return_value='i-am-a-public-key')
+        return session
+
+
+class RS256HTTPSPublicKeyRetrieverTest(utils.RS256KeyTestMixin, TestCase):
```

Comment:
Will do. 

For record: subclassing utils.RS256KeyTestMixin and TestCase allows code completions in IDE for classes' methods and eliminates IDE warnings like this:

> Unresolved attribute reference 'assertEqual' for class 'BaseHTTPSPublicKeyRetrieverTestMixin'